canonical-web-and-design/ubuntu.com
# Add K8s 1.21 to release-cycle

## Done

Add 1.21 Kubernetes to release-cycle page/chart

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- check chart on /about/release-cycle page

![Screenshot from 2021-04-09 13-44-03](https://user-images.githubusercontent.com/115290/114181731-c4ddfc00-9939-11eb-924d-a6243d4c1888.png)
